### PR TITLE
Quality/103/code smells

### PR DESCRIPTION
**Pull request by finndavidson**
fixed code smells and bugs picked up by SonarQube

### CODE REVIEW

File: app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java [hunk 1]
```diff
@@ -72,7 +71,7 @@ public void onCameraChange() {
 
             LatLngBounds bounds = mMap.getProjection().getVisibleRegion().latLngBounds;
 
-            if (mMap.getCameraPosition().zoom > mZoomLevel) {
+            if (mMap.getCameraPosition().zoom > ZOOMLEVEL) {
```
Comment (patnunes) [inline app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java]: should this be ZOOM_LEVEL to match our conventions?

File: app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java [hunk 2]
```diff
@@ -33,7 +32,7 @@
 
 public class MapInitializer {
 
-    public static final double mZoomLevel = 18.3;
+    public static final double ZOOMLEVEL = 18.3;
```
Comment (patnunes) [inline app/src/main/java/com/conupods/OutdoorMaps/MapInitializer.java]: should this be ZOOM_LEVEL to match our conventions?

File: app/src/debug/res/values/google_maps_api.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <resources>
     <!--
-    TODO: Before you run your application, you need a Google Maps API key.
+    DONE: Before you run your application, you need a Google Maps API key.
```
Comment (MinaSelim) [inline app/src/debug/res/values/google_maps_api.xml]: HAHAAHAHAHAHA

File: app/src/main/java/com/conupods/IndoorMaps/ConcreteBuildings/HBuilding.java [hunk 4]
```diff
@@ -14,7 +13,7 @@
     private static Map<LatLng, Building> mDataMapHash = mDataInstance.getDataMap();
     private static Building instance;
 
-    private HBuilding(String campus, String code, String name, String longName, String address, LatLng latLng, LatLng overlayLatLng, List<String> classRooms) {
+    public HBuilding(String code, String name, String longName, String address, LatLng latLng, LatLng overlayLatLng, List<String> classRooms) {
```
Comment (MinaSelim) [inline app/src/main/java/com/conupods/IndoorMaps/ConcreteBuildings/HBuilding.java]: make pribate, singleton pattern

File: app/src/main/java/com/conupods/IndoorMaps/ConcreteBuildings/MBBuilding.java [hunk 5]
```diff
@@ -16,7 +14,7 @@
     private static Map<LatLng, Building> mDataMapHash = mDataInstance.getDataMap();
     private static Building instance;
 
-    private MBBuilding(String campus, String code, String name, String longName, String address, LatLng latLng, LatLng overlayLatLng, List<String> classRooms) {
+    public MBBuilding(String code, String name, String longName, String address, LatLng latLng, LatLng overlayLatLng, List<String> classRooms) {
```
Comment (MinaSelim) [inline app/src/main/java/com/conupods/IndoorMaps/ConcreteBuildings/MBBuilding.java]: make private, singleton pattern

File: app/src/main/java/com/conupods/IndoorMaps/ConcreteBuildings/VLBuilding.java [hunk 6]
```diff
@@ -15,7 +14,7 @@
     private static Map<LatLng, Building> mDataMapHash = mDataInstance.getDataMap();
     private static Building instance;
 
-    private VLBuilding(String campus, String code, String name, String longName, String address, LatLng latLng, LatLng overlayLatLng, List<String> classRooms) {
+    public VLBuilding(String code, String name, String longName, String address, LatLng latLng, LatLng overlayLatLng, List<String> classRooms) {
```
Comment (MinaSelim) [inline app/src/main/java/com/conupods/IndoorMaps/ConcreteBuildings/VLBuilding.java]: make private, singleton pattern

**Pull request merged**